### DISCUSSION TITLE
Is it possible to customise / suppress theorem numbering?

### DISCUSSION THREAD

**Discussion by aaronsgithub** [quarto-dev/quarto-cli#5479] [Q&A]
I'm looking for ways to achieve the following:

- have an unnumbered theorem
- have theorems, definitions etc have continuation of numbering e.g. Theorem 1, Definition 2 etc
- have section numbering for theorems e.g. Theorem 1.2

Any tips would be much appreciated : )

**Comment by cscheid** [comment]
Not easily as of quarto 1.3. I'm working on crossrefs and the situation should be improved when 1.4 releases.

**Reply by aaronsgithub** [reply]
Nice. Thank you for your response and for your work in Quarto.

**Comment by ute** [comment]
I had the same request and couldn't wait. 
Therefore I wrote an extension that does exactly this : [custom-numbered-blocks](https://github.com/ute/custom-numbered-blocks)

Here here is a very minimal example:
![image](https://github.com/quarto-dev/quarto-cli/assets/5145859/83e0ceea-6e26-4834-b10c-3b2efb31a033)

Quarto is simply awesome :-)

**Comment by mgreenbe** [comment]
For an unnumbered theorem, I think the natural annotation would be simply
```
::: {.thm}
content
:::
```
This would just apply the `thm` related `css` class (maybe it's even called `.thm`, or some namespaced variant) without wiring the theorem into the cross-reference structure. Is this too naïve?

**Reply by mcanouil** [reply]
There is no such class in Quarto.

FYI, what [Carlos mentioned](https://github.com/quarto-dev/quarto-cli/discussions/5479#discussioncomment-5861293) is the custom cross-reference: https://quarto.org/docs/authoring/cross-references-custom.html
